Brain/ChatBot.py

```
@@ -5,12 +5,13 @@
 import json
 from groq import Groq
 from Backend import TTS
+from Backend.RealtimeData import RealTimeInformation
 
 # Load environment variables from .env
 load_dotenv(dotenv_path='../.env')
 
 class Chatbot:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)